AdrianBarros/assignments/treesBFS/lc102/lc102.py

```diff
@@ -0,0 +1,117 @@
+# Problem Statement #
+
+# Given a binary tree, populate an array to represent its level-by-level traversal. You should populate the values of all nodes of each level from left to right in separate sub-arrays.
+
+
+class TreeNode:
+    def __init__(self, val):
+        self.val = val
+        self.left, self.right = None, None
+
+
+'''
+input: bi tree
+output: an array of arrays of all the node lv/lv left to right 
+
+step1: traverse each level of the tree
+step2: for each node well keep track each in a queue
+step3: if the whole lv its inside the queue, pushing the next from the left
+        added step: pop the respective node inside an array 
+step4: every time we loop we are going to make another loop to keep track of the inner arrays
+step5: push a currect array into the result and repeat
+final return result array
+'''
+
+
+def traverse(root):
+    result = []
+    # TODO: Write your code here
+    queue = []
+    queue.append(root)
+    while len(queue) > 0:
+        lv = []
+        counter = len(queue)
+        # get to add the correct level inside of a inside array
```

Comment:
remove single line comments from programming in interviews always before submitting